public/index.html

```diff
@@ -23,7 +23,7 @@
     	<script src="/scripts/imageSliderScript.js"></script>
     	<script src="/scripts/imageGridScript.js"></script>
     	<script src="/scripts/timeline.js?version=1005"></script>
-	<script src="/scripts/histogram.js"></script>
+		<script src="/scripts/histogram.js"></script>

```

Comment:
This indentation is unnecessary.

```suggestion
	<script src="/scripts/histogram.js"></script>
```